packages/twenty-server/src/engine/core-modules/jwt/services/jwt-wrapper.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService, JwtSignOptions, JwtVerifyOptions } from '@nestjs/jwt';
+
+import * as jwt from 'jsonwebtoken';
+
+@Injectable()
+export class JwtWrapperService {
+  constructor(private readonly jwtService: JwtService) {}
+
+  sign(payload: Buffer | object, options?: JwtSignOptions): string;
```

Comment:
providing same interfaces but I feel we could simplify more, all these options are overkill

Reply:
actually I have simplified :D